approval-decision-utils-product/README.md

```diff
@@ -0,0 +1,237 @@
+
+# Approval Decision Utils
+  
+This component provides a standardized approach for implementing the approval flow. Users can make decisions, confirmations, leave comments, and view the approval history in a data table.
```

Comment:
Let's pimp this text a bit up (marketing):
Axon Ivy's **Approval Decision** Utils provides a standardized approach for implementing your approvals in any business process.  This component:

- Gives you a standard pattern to make decisions and confirmations
- Implements a comment function for better documentation
- Provides a clear view of the ongoing approval history to meet regulatory, compliance, and transparency requirements
- Supports you with an easy-to-copy demo implementation to reduce your integration effort.